Source/WebCore/dom/Document.cpp

```diff
@@ -3404,12 +3404,12 @@ void Document::implicitClose()
 
 void Document::setParsing(bool b)
```

Comment:
We should rename this setter to `setIsParsing` and the parameter to `isParsing`.